package.json

```diff
@@ -13,9 +13,10 @@
   "author": "Julie Ralph <ju.ralph@gmail.com>",
   "dependencies": {
     "adm-zip": "0.4.7",
+    "chalk": "^1.1.3",
```

Comment:
Chalk has a non-trivial dependency tree:

`
└─┬ chalk@1.1.3
  ├── ansi-styles@2.2.1
  ├── escape-string-regexp@1.0.5
  ├─┬ has-ansi@2.0.0
  │ └── ansi-regex@2.0.0
  ├── strip-ansi@3.0.1
  └── supports-color@2.0.0
`

which is a pain for import. Can we make it optional? Just something like trying to require, catching an error and just returning a plain string if not available.